### DISCUSSION TITLE
Game crash in multiplayer when downloading multiple beatmaps at the same time

### DISCUSSION THREAD

**Discussion by mty22** [ppy/osu#19183] [Q&A] [answered]
lazer build: 2022.716.1-lazer

actions:

1. join existing multi lobby
2. set difficulty change -> AR to 10
3. click download on 3 queued maps within a second

game locked up for about 10-15 seconds (audio ceased to play, no mouse movement etc) -> crashed shortly after.

logs/eventvwr logs attached.

[mp_lobby_crash.zip](https://github.com/ppy/osu/files/9127404/mp_lobby_crash.zip)

`2022-07-17 08:32:58` within runtime log is when the crash occured. 

twitch: frznmatt
osu profile: https://osu.ppy.sh/users/13298271  (frzn)

**Comment by frenzibyte** [comment]
As a side note, feel free to open an issue thread next time if you have all the necessary info for tracking this, which you have here (build version, steps, logs).

**Reply by mty22** [reply]
Ah, last time I did that @frenzibyte.. I was advised to create a discussion instead (as it can be promoted into an issue thread later).

**Reply by frenzibyte** [reply]
I think that was before we started allowing issue threads with a proper bug report form.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/19211
